### PR TITLE
new react webpart for Halloween 🎃🎃🎃 (react-find-parker)

### PR DESCRIPTION
**Pull request by Adam-it**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no - yes?                               |
| New feature?    | no - yes?                               |
| New sample?     | yes?                               |
| Related issues? | fixes #X, partially #Y, mentioned in #Z |

## What's in this Pull Request?

hi 👋,

I would like to submit a very simple new react webpart created for an Halloween 🎃🎃🎃 event. But the webpart may be used for any other event like easter (to find easter eggs I guess 🤔). 
The webpart is a simple find elements game. We set in property pane how many elements we want to find, then we start, find all the objects, and... and that's it 😉. Nothing fancy, very simple, a little fun 👍👍

This is how it looks like by default we look for PnP Parkers 😉
parker

When changing the svg icon we may create a Halloween version

halloween

or where's Wally version
findWally

### CODE REVIEW

File: samples/react-find-parker/src/webparts/findParker/components/FindParker/FindParker.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    const listOfParkers = this.createListOfParkers(numberOfElements);
+    this.setState({
+      listOfParkers: listOfParkers,
+      numberOfFoundElements: 0
+    });
+    this.renderParkers(listOfParkers);
+  }
+
+  private createListOfParkers(numberOfElements: number): Array<IParker> {
+    const pageHeight: number = document.querySelector(Constants.mainPageContent).scrollHeight;
```
Comment (hugoabernier) [inline samples/react-find-parker/src/webparts/findParker/components/FindParker/FindParker.tsx:123]: You may want to check that the placeholder is available. In the workbench, these next two lines will generate errors and the rest of this function will fail.

Comment (Adam-it) [reply samples/react-find-parker/src/webparts/findParker/components/FindParker/FindParker.tsx:140]: ok I added a double check in `componentDidMount` for this div and if it is not present I show a message to the user that something is not ok and he or she may not start the game.
this is how it looks/behaves now on workbench
image

and this is on a SharePoint Site
image

**Comment by Adam-it** [comment]
@hugoabernier all resolved 👍
thank you very much for finding this 👍👍 and for all the effort in double checking the PR 🙏🙏.
If there is anything more I should investigate or do please let me know 👍

**Comment by hugoabernier** [comment]
Thank you @Adam-it! I love the message on the workbench!

**Pull request merged**